docs/podman-run.1.md

```diff
@@ -790,11 +812,7 @@ You would have to write policy defining a `svirt_apache_t` type.
 If you want to set `/dev/sda` device weight to `200`, you can specify the device
 weight by `--blkio-weight-device` flag. Use the following command:
 
-   # podman run -it --blkio-weight-device "/dev/sda:200" ubuntu
-
-```
-$ podman run -d busybox top
```

Comment:
Was this example intentionally dropped?  It's originally from 8cf07b2ad1 (#2) in case that helps.

Reply:
It didn't look like it fit in with the `--blkio-weight-device` example.

I could be misunderstanding how that example works, though.

Reply:
@wking I was OK removing it as I didn't see how it applied to the vebiage above is at line 793 does.  @mheon @baude any thoughts?